en/docs/Learn/APISecurity/Authorization/role-based-access-control-using-xacml.md

```diff
@@ -47,42 +47,57 @@ Based on the requirement, a single API is exposed to add or retrieve order info
 
 5.  Start the WSO2 Identity Server and log in to its Admin Console.
 
-        !!! tip
-    Since API Manager and Identity Server run on the same server, offset the Identity Server by 1.
+    !!! tip
+        Since API Manager and Identity Server run on the same server, offset the Identity Server by 1.
 
 
 6.  Under the **Entitlement** section, click **Policy Administration &gt; Add New Entitlement Policy** .
     ![]({{base_path}}/assets/attachments/103334839/103334836.png)
     ![]({{base_path}}/assets/attachments/103334839/103334835.png)
-7.  You are redirected to a page listing all available policy editors. Select **Standard Policy Editor** from the list and add the values shown below in the policy editor. Refer [Creating a XACML Policy](https://docs.wso2.com/display/IS570/Creating+a+XACML+Policy) in WSO2 Identity Server for more information.
+7.  You are redirected to a page listing all available policy editors. Select **Standard Policy Editor** from the list and add the values shown below in the policy editor. Refer [Creating a XACML Policy](https://is.docs.wso2.com/en/5.9.0/learn/creating-a-xacml-policy/) in WSO2 Identity Server for more information.
     1.  **Entitlement Policy Name:** PizzaShackPolicy
     2.  **Rule Combining Algorithm:** Deny unless Permit
 
     When the rule combination algorithm is set to **Deny Unless Permit** , you need to set the permit criteria as a rule.
+    
 8.  In the **Define Entitlement Rule(s)** area, set the following 2 rules to define the kind of requests and from which user they should be permitted.
     1.  AdminGrant - grants full access to the admin user. Give the information below,
+
         **Rule Name:** AdminGrant
+        
         **Conditions:** Under **Define your conditions by using followings....** , select drop down options as **Subject** , **is/are** , **at-least-one-member-of** in order and enter **admin** in the last field.
         Click the icon next to **END** shown below to configure the attribute value and attribute source to retrieve the user roles from the user store.
         ![]({{base_path}}/assets/attachments/103334839/103334834.png)
```

Comment:
The images need to be moved out of the {{base_path}}/assets/attachments folder and added in the {{base_path}}/assets/img folder based on where it should be located (e.g., If the doc is under the learn section then add it in the /learn folder
The image names need to be changed to a meaningful name.

Reply:
fixed with a8f029f